README.md

```diff
@@ -243,7 +243,7 @@ Become a sponsor and get your logo on our README on GitHub with a link to your s
 
 ## Changelog
 
-RuboCop's changelog is available [here](CHANGELOG.md).
+RuboCop's changelog is available [here](https://github.com/rubocop/rubocop/releases).
```

Comment:
I'd probably change the section headings here and below to "Release Notes" and drop the changelog term, so it's not confusing for the users.